wandb/integration/keras_core/metrics_logger.py

```diff
@@ -0,0 +1,155 @@
+import sys
+from typing import Any, Dict, Optional, Union
+
+import keras_core as keras
+from keras_core.callbacks import Callback
```

Comment:
Maybe, do it under a try-except clause with a user friendly warning in case keras_core is not available.